### PR TITLE
Add benches

### LINKED ISSUE #830: Performance
How does the performance of RustPython compare (or will compare) with CPython or PyPy?

### PR DESCRIPTION
**Pull request by LuoZijun**
Resolve #830

### CODE REVIEW

**Comment by LuoZijun** [comment]
```bash
cargo bench
```
Output:

```
     Running target/release/deps/parser-9219a509bec6327a

running 2 tests
test bench_parse_to_ast ... bench:  60,803,145 ns/iter (+/- 4,723,694) = 1 MB/s
test bench_tokenization ... bench:  49,881,125 ns/iter (+/- 2,945,707) = 1 MB/s

test result: ok. 0 passed; 0 failed; 0 ignored; 2 measured; 0 filtered out

```

**Comment by windelbouwman** [comment]
Did you see this folder? https://github.com/RustPython/RustPython/tree/master/benchmarks

**Comment by LuoZijun** [comment]
@windelbouwman 

Yeah, i know that folder. if you decided to use Python to write benchmark, So how do we write some internal benchmarks? e.g lexer, parser.

**Comment by windelbouwman** [comment]
Okay, that is good. Could you move the benchmarks into the benchmarking folder so that we do not duplicate the nbody.py and mandelbrot.py files? Also, we would have a single benchmarking directory with multiple types of benchmarks.

File: data/nbody.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
+
+# The Computer Language Benchmarks Game
+# https://salsa.debian.org/benchmarksgame-team/benchmarksgame/
+#
+# originally by Kevin Carson
+# modified by Tupteq, Fredrik Johansson, and Daniel Nanz
```
Comment (windelbouwman) [inline data/nbody.py]: Please use the file available in `benchmarks/benchmarks`.

File: data/minidom.py [hunk 2]
```diff
@@ -0,0 +1,1981 @@
+"""Simple implementation of the Level 1 DOM.
```
Comment (windelbouwman) [inline data/minidom.py]: Please move this file to `benchmarks/benchmarks`

File: benches/vm.rs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#![feature(test)]
+
+extern crate rustpython_parser;
+extern crate rustpython_vm;
+extern crate test;
```
Comment (windelbouwman) [inline benches/vm.rs]: Please move this file to the `benchmarks` folder.

**Comment by LuoZijun** [comment]
@windelbouwman 

I turned off the stdout of `nbody.py` and `mandelbrot.py` files. 
because they were not necessary for a benchmark.

File: benchmarks/bench.rs [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+    let gil = cpython::Python::acquire_gil();
+    let python = gil.python();
+
+    let globals = None;
+    let locals = None;
+
+    b.iter(|| {
+        let res: cpython::PyResult<()> = python.run(source, globals, locals);
+        assert_eq!(res.is_ok(), true);
```
Comment (coolreader18) [inline benchmarks/bench.rs]: This can just be `assert!(res.is_ok())`

Comment (LuoZijun) [reply benchmarks/bench.rs]: ok.

**Pull request merged**